common/src/test/scala/uk/ac/wellcome/models/ItemTest.scala

```diff
@@ -23,13 +24,13 @@ class ItemTest extends FunSpec with Matchers with JsonTestUtil {
       |  "locations": [
       |    {
       |      "locationType": "location",
+      |      "url" : null,
+      |      "credit" : null,
       |      "license": {
       |        "licenseType": "${License_CCBY.licenseType}",
       |        "label": "${License_CCBY.label}",
-      |        "url": "${License_CCBY.url}",
-      |        "type": "License"
-      |      },
-      |      "type": "Location"
+      |        "url": "${License_CCBY.url}"
+      |      }
```

Comment:
I only changed `Work` and `Item` to have an `ontlogyType` field that can be serialised by circe. That is because they are `Identifiable` and the id_minter needs the `ontologyType` to save in the database along with the `sourceIdentifier`. 
For other types, I don't actually think we need this. It's more relayed to how we display things in the API, so I would be in favour of removing all of these types out of our internal data model. What do you think?

Reply:
As discussed, I think we do need to pass these types around internally.